source/scripts/ev3dev-config.sh

```diff
@@ -265,6 +265,11 @@ do_hardware_menu() {
   current_platform=$(detect_current_platform)
   get_available_hardware_options "$current_platform"
   
+  if [ ${#HARDWARE_MENU_OPTIONS[@]} -eq 0 ]; then
+    whiptail --msgbox "Sorry, there are no available hardware options for your platform." 20 60 1
```

Comment:
No need to apologize. We could just say "There are currently no hardware options for this device."